README.md

```diff
@@ -414,19 +414,17 @@ set PATH=C:\STL\out\x64\out\bin\amd64;%PATH%
 You don't need any extra steps to run with test code and the code in STL headers instrumented with [ASan][].
 The test matrices include both ASan and non-ASan configurations.
 
-However, to instrument the separately-compiled code (the DLL, the satellites, the [Import Library][] - everything that's
-in `/stl/src`), you need to build the STL with ASan. Change the build steps to add `-DSTL_ASAN_BUILD=ON`:
+However, to instrument the separately-compiled code (the DLL, the satellites, the [Import Library][] - everything that's in `/stl/src`), you need to build the STL with ASan. Change the build steps to add `-DSTL_ASAN_BUILD=ON`:

```

Comment:
The lone wrapping was deliberate and should be restored to make code validation pass

Reply:
We enforce a 120-column maximum line length on most files in the STL. You can build the `validate` target to be yelled at for various violations of rules we enforce on file contents.

Reply:
Oh, interesting. Good to know - I've undone these changes.